DataParallelTable.lua

```diff
@@ -198,8 +198,12 @@ function DataParallelTable:__init(dimension)
    self.gradOutputGpu = {} -- gradOutputs for each gpu
    self.outputGpu = {} -- outputs for each gpu
    self.gradInputGpu = {} -- gradInput for each gpu
+   self.flattenParams = true 
```

Comment:
Is this supposed to be always true ? If yes  we can take it off from the constructor, right ?